app/main.ipynb

```diff
@@ -126,19 +126,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "\n",
+    "subsequence = array[[0, 1, 2, 4, 6]]\n",
+    "\n",
+    "result = subsequence.sum()"
```

Comment:
The subsequence selected does not sum to 27. Please check the indices used for selection and adjust them to achieve the desired sum.